pkg/asset/agent/image/unconfigured_ignition.go

```diff
@@ -185,24 +185,26 @@ func (a *UnconfiguredIgnition) Generate(_ context.Context, dependencies asset.Pa
 		// Enable the agent-check-config-image.service for the current workflow.
 		enabledServices = append(enabledServices, "agent-check-config-image.service")
 
-	case workflow.AgentWorkflowTypeInstallInteractiveDisconnected:
-		// Add the rendezvous host file. Agent TUI will interact with that file in case
-		// the rendezvous IP wasn't previously configured, by managing it as a template file.
-		if rendezvousIP == "" {
-			rendezvousHostFile := ignition.FileFromString(rendezvousHostEnvPath, "root", 0644, rendezvousHostTemplateData)
-			config.Storage.Files = append(config.Storage.Files, rendezvousHostFile)
+		// To Do:
+		if sentinelfiel is present or another way to identify we want to invoke AgentWorkflowTypeInstallInteractiveDisconnected workflow {
+			// Add the rendezvous host file. Agent TUI will interact with that file in case
+			// the rendezvous IP wasn't previously configured, by managing it as a template file.
+			if rendezvousIP == "" {
+				rendezvousHostFile := ignition.FileFromString(rendezvousHostEnvPath, "root", 0644, rendezvousHostTemplateData)
+				config.Storage.Files = append(config.Storage.Files, rendezvousHostFile)
+			}
+
+			// Explicitly disable the load-config-iso service, not required in the current flow
+			// (even though disabled by default, the udev rule may require it).
+			config.Storage.Files = append(config.Storage.Files, ignition.FileFromString("/etc/assisted/no-config-image", "root", 0644, ""))
+
+			// Enable the UI service.
+			interactiveUIFile := ignition.FileFromString("/etc/assisted/interactive-ui", "root", 0644, "")
+			config.Storage.Files = append(config.Storage.Files, interactiveUIFile)
```

Comment:
This sentinel file will be added by the invoker , appliance or saas

Reply:
example in appliance PR https://github.com/openshift/appliance/pull/548/files#diff-881ebdc27674fa156f1d36f5a92ade6d7b78a08c431e9a352611f59f97c3713bR78-R79